From b15839281b159c8b641c7a86f5a08c00913d0b31 Mon Sep 17 00:00:00 2001
From: Ruediger Stock <ruediger.stock@intel.com>
Date: Tue, 12 Feb 2019 09:16:54 +0100
Subject: [PATCH 108/120] ASoC: Intel: Skylake: Firmware reload alignment

partially revert d95b897d1f80f1d5bd8f6e9e0c3654422072df53
setting BXT_FW_INIT_RETRY back to 100 to address rtc 284994
---
 sound/soc/intel/skylake/bxt-sst.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/intel/skylake/bxt-sst.c b/sound/soc/intel/skylake/bxt-sst.c
index 6d823bf..dd56a63 100755
--- a/sound/soc/intel/skylake/bxt-sst.c
+++ b/sound/soc/intel/skylake/bxt-sst.c
@@ -59,7 +59,7 @@
 /* Delay before scheduling D0i3 entry */
 #define BXT_D0I3_DELAY 5000
 
-#define BXT_FW_INIT_RETRY 3
+#define BXT_FW_INIT_RETRY 100
 
 #define GET_SSP_BASE(N)	(N > 4 ? 0x2000 : 0x4000)
 
-- 
1.9.1

